spec/Dynamic-Validator.md

```diff
@@ -230,17 +234,41 @@ The type of the messages depends on the consensus engine. For example, type Mess
 stakeholders = [ address+ ], address asc
 balance(address) = quantity
 delegation(delegator) = [ [delegatee, quantity]+ ], delegatee asc
-candidates = [ [pubkey, deposits, nominate_end_at, metadata]+ ], pubkey asc
+candidates = [ [pubkey, deposits, nominate_end_at, metadata]+ ], 'priority' asc. See candidate prioritizing
 banned = [ address+ ], address asc
 jailed = [ [address, deposits, custody_until, released_at]+ ], address asc
 term_id = [ the last block number of the previous term, the current term id ]
 intermediate_rewards = [ [ address, rewards ]+ address asc, [ address, rewards ]+ address asc ]
 validators = [ [ weight, delegation, deposit, pubkey ] ] (weight, delegation, deposit, pubkey) asc
 ```
 
+### Candidate prioritizing
+
+We elect validators based on the delegated CCS quantity and deposit amount.
+However, there may be a tie with the same delegations and deposit amount.
+To break a tie, we give priority to the candidate who have responded most recently.
+
+Current validators will have highest priority among candidates with the same `(delegation, deposit)` at the next election.
+The sender of most recent `SelfNominate` transaction will be after them.
+
+Thus, We repriortize candidates on `SelfNomination` transaction and `TermEnd` event with following algorithm.
```

Comment:
Thus, we reprioritize candidates on `SelfNomination` transaction and `TermEnd` event with the following algorithm.